tests/views_test.py

```diff
@@ -1,5 +1,100 @@
-from src_api.views import helloworld
+from numpy import ndarray, result_type
+from src_api.views import *
+import shutil
+import os
 
+def TestBasics():
+    """
+    Sets the Basic Folder/Files for Testing.
 
-def test_helloworld():
-    assert helloworld() == "Hello world !"
+    Returns:
+        tuple: filenames and folder.
+    """
+    folder_name = os.path.join(app.config["APP_FOLDER"], "tests", "testfiles")
+    file_name = "test_file.wav"
+    file_name_copy = "test_file_copy.wav"
+    return folder_name,file_name,file_name_copy
+
+
+def TestBasicsCopy(shutil):
+    """
+    Creates Copies of the Files from TestBasics() for Testing.
+    
+    Returns:
+        tuple:  filenames and folder.
+    
+    """
+    folder_name, file_name, file_name_copy = TestBasics()
+    shutil.copyfile(os.path.join(folder_name,file_name), os.path.join(folder_name,file_name_copy))
+    return folder_name,file_name_copy
+
+
+def test_predict_from_file():
+    print()
+
+
+def test_backend_save_request_object():
+    """
+    filetype?
+    """
+    print()
+
+
+def test_backend_rawfile_load():
+    print()
+    
+
+def test_backend_file_load():
+    print()
+
+
+def test_get_duration():
+    print()
+
+
+def test_get_sampling_rate():
+    """
+    Checks if the the Returned Sampling Rate of def_sampling_rate is 8000.
+    """
+    
+    folder_name, file_name_copy = TestBasicsCopy(shutil)
+    result = get_sampling_rate(file_name_copy,folder_name)
+    assert result == 8000
+
+
+def test_load_audio_sequence():
```

Comment:
Same comment as above.
This function to return a sequence (numpy array) of float based on a specified sampling rate and max array dimension so one can use paramtrization to try multiple configurations and test if the output is always of the specified shape (not necessarily (8000,)